wiki/en/Running-a-Server.md

```diff
@@ -179,131 +261,50 @@ Run `sudo systemctl daemon-reload` to register them for first use.
-
-### Recording Directory
-
-This sets the path to where the Server's recordings will be stored. With this path set, the "Enable Jam Recorder" function in the Server Setup tab will make recording start once the first person connects to the Server, and stops when the last person leaves. Use the "New Recording" button to create a new sub-directory in which the recordings will be stored from then on. Note that Recordings are per track in [Audacity](https://www.audacityteam.org/) `.lof` format and [REAPER](https://en.wikipedia.org/wiki/REAPER) `.rpp`. Open the respective files to listen to them in those applications.
-
-**Note**: When your Server is recording, Clients will display a message that recording is on.
-
-### Custom Directory Address
-
-Leave this field empty unless you need to list your Server on a [Custom Directory](#3-directory) or run a Directory.
+* **Linux users** - Launch the "Jamulus Server" shortcut. Or you can open a terminal window (`CTRL+ALT+t` on Debian and related distros), type `jamulus -s` and hit return.
 
-### Server List Filename
+While most common functions in Jamulus can be set using the GUI, others can only be set using options given in a terminal window when the server is launched. Exactly how you do this will depend on your operating system.
 
-Leave this field empty unless you need to run your Server as a [Directory](#3-directory). When in use, this holds the list of registered Servers whilst restarting the Directory. This prevents the server list appearing "empty" until the Servers re-register.
+For example on Windows, to use a specific settings file, right-click on the Jamulus shortcut and choose “Properties” > Target. Add the necessary arguments to Jamulus.exe:
 
-### Delay panning
+ `"C:\Program Files\Jamulus\Jamulus.exe" --serverbindip 192.168.0.100`
 
-This option uses small differences in sound arrival time between the two ears. It produces a stereo effect similar to natural human hearing when compared to normal "volume" panning.
+For macOS, start a Terminal window and run Jamulus with the desired options like this:
 
-### Start Minimised
+ `/Applications/Jamulus.app/Contents/MacOS/Jamulus --serverbindip 192.168.0.100`
 
-**Windows users** - If you want the Server to start automatically on system start, enable the corresponding check box.
+**Note** Command line options will set the Server’s defaults at startup. You can override them with their corresponding GUI controls while the Server is running.
 
-### Server status icon
+### The Server status icon
 
-When the Server is running, the operating system will show an icon in the system tray or status area to represent whether the Server is active:  
+When a Server is running in GUI mode, the operating system will show an icon in the system tray or status area to represent whether the Server is active:  
```

Comment:
I can't see these images
```suggestion
When a Server is running in GUI mode, the operating system will show an icon in the system tray or status area that indicates whether the Server is active:  
```

Reply:
Have you built the site locally? Github won't show them to you in context here.

Reply:
Do you use Sphinx for a local build?

Reply:
Not as far as I know. You can [use these instructions](https://github.com/jamulussoftware/jamuluswebsite#viewing-the-website-offline).